### ISSUE TITLE
Casa Matriz: Indicadores de rendimiento #5

### ISSUE THREAD

**Issue by CrisRoa** [axelbol/rfp-1#24]
¿Estos indicadores solamente serán vistos por la casa matriz?

**Comment by axelbol** [comment]
No, deberán poder ser vistos por la casa matriz y por las sucursales pero con sus respectivas limitaciones para las sucursales ya que estas solo deberán poder visualizar los indicadores de rendimiento de su respectiva tienda, la tienda 'x' debe poder visualizar indicadores de rendimiento de la tienda 'x'.
En cambio la casa matriz podrá visualizar indicadores de rendimiento de las tres sucursales, es decir, deberá exisitir la opción para poder elegir una de las tres sucursales y ver los indicadores de esta tienda.
